front/app/containers/Admin/projects/project/analysis/Insights/ReferenceLink.tsx

```diff
@@ -31,6 +31,39 @@ const StyledLink = styled(Link)<{ isActive: boolean }>`
     }`}
 `;
 
+const referenceDisplayValue = (input, customField, localize) => {
```

Comment:
I can't use `useLocalize()` inside a normal (?) function? Is there a better way?

Reply:
There are a couple ways to do this - either you pass localize as you've done, or you can call localize on the result of referenceDisplayValue() in the component where you use it instead :) Either is alright I think! I have a slight preference for the second option though.